src/main/java/team/themoment/hellogsmv3/domain/application/service/ModifyApplicationService.java

```diff
@@ -0,0 +1,269 @@
+package team.themoment.hellogsmv3.domain.application.service;
+
+import jakarta.persistence.EntityManager;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import team.themoment.hellogsmv3.domain.applicant.entity.Applicant;
+import team.themoment.hellogsmv3.domain.applicant.repo.ApplicantRepository;
+import team.themoment.hellogsmv3.domain.application.dto.request.ApplicationReqDto;
+import team.themoment.hellogsmv3.domain.application.entity.*;
+import team.themoment.hellogsmv3.domain.application.entity.abs.AbstractApplication;
+import team.themoment.hellogsmv3.domain.application.entity.param.AbstractApplicationStatusParameter;
+import team.themoment.hellogsmv3.domain.application.entity.param.AbstractPersonalInformationParameter;
+import team.themoment.hellogsmv3.domain.application.entity.param.CandidateMiddleSchoolGradeParameter;
+import team.themoment.hellogsmv3.domain.application.repo.ApplicationRepository;
+import team.themoment.hellogsmv3.domain.application.type.DesiredMajors;
+import team.themoment.hellogsmv3.domain.application.type.GraduationStatus;
+import team.themoment.hellogsmv3.domain.application.type.MiddleSchoolTranscript;
+import team.themoment.hellogsmv3.domain.auth.repo.AuthenticationRepository;
+import team.themoment.hellogsmv3.global.exception.error.ExpectedException;
+
+import java.math.BigDecimal;
+
+@Service
+@RequiredArgsConstructor
+public class ModifyApplicationService {
+
+    private final ApplicantRepository applicantRepository;
+    private final AuthenticationRepository authenticationRepository;
+    private final ApplicationRepository applicationRepository;
+    private final EntityManager em;
+
+    @Transactional
+    public void execute(ApplicationReqDto dto, Long userId, boolean isAdmin) {
+
+        Applicant applicant = applicantRepository.findById(userId)
+                .orElseThrow(() -> new ExpectedException("존재하지 않는 유저입니다.", HttpStatus.NOT_FOUND));
```

Comment:
ApplicantService 같은 서비스 객체에서 
Applicant를 찾고 없다면 예외를 던지는 method를 정의하고 사용하는게 좋을 것 같습니다.

```suggestion
applicantService.findOrThrow(userId);
```

Reply:
76beb0c803a7e821ff65ab17c1693e47a8c750f6

의견 감사드립니다. 해당 메서드를 구현하여 `Applicant` 객체를 조회하는 로직에 모두 적용시켰습니다.